APCAppCore/APCAppCore/UI/TableViewCells/APCFeedTableViewCell.m

```diff
@@ -51,10 +51,12 @@ - (void)setupAppearanceForRead:(BOOL)read
         self.titleLabel.font = [UIFont appRegularFontWithSize:19.0f];
         self.descriptionLabel.font = [UIFont appRegularFontWithSize:16.0f];
         self.dateLabel.font = [UIFont appRegularFontWithSize:14.0f];
+        self.backgroundColor = [UIColor colorWithRed:0.97 green:0.97 blue:0.97 alpha:1];
```

Comment:
Please use UIColor+APCAppearance.h colors, adding to this if it makes sense. Do not hardcode.